packages/runtime/src/interpreter/opcode.ts

```diff
@@ -6,88 +6,152 @@ import type { TEALStack } from "../types";
 import { BIGINT0, BIGINT1 } from "./opcode-list";
 
 export class Op {
-  assertMinStackLen (stack: TEALStack, minLen: number): void {
+  /**
+   * assert stack length is less than minLen
```

Comment:
```suggestion
   * assert stack length is atleast minLen
```

Reply:
atleast would mean we want stack to be greater than minLen

Reply:
we cannot use `atmost` here too, because that will include minLen

Reply:
> atleast would mean we want stack to be greater than minLen

Isn't this what we want? . `assertMinStackLen(stack, 2)` means that there should be atleast 2 elements in stack (probably because the `opcode` is looking to pop two elements. Am i missing something?

Reply:
hmm, i was thinking of something else here, you are right